### DISCUSSION TITLE
"The repository is owned by a group." But it is my personal repo

### DISCUSSION THREAD

**Discussion by mitskelevychp** [vercel/community#6794] [Account Issue]
### Question

Hi )
I received the message: "The repository "my/repo-name" is owned by a group. Please link it to a Vercel Pro team instead."
And I can't create a project.
But I am not part of the group, this is my personal repository. And I have only 2 projects on Vercel. What can I do? I just write a code for myself and I need a free account for my project.
I will be grateful for your help!

**Comment by amyegan** [comment]
Hey @mitskelevychp. Is the repo part of a GitLab Group, GitHub Organization, or Bitbucket Project?

https://vercel.com/docs/deployments/git#deploying-private-git-repositories
